app/src/main/java/com/github/polypoly/app/menu/WelcomeActivity.kt

```diff
@@ -160,7 +173,9 @@ class WelcomeActivity : ComponentActivity() {
             ) {
                 Surface(
                     color = MaterialTheme.colors.primary,
-                    modifier = Modifier.fillMaxWidth(0.95f).fillMaxHeight(0.95f)
+                    modifier = Modifier
+                        .fillMaxWidth(0.95f)
+                        .fillMaxHeight(0.95f)
```

Comment:
Magic numbers should be avoided

Reply:
I also agree with that for magic numbers. But in this case these are just parameters used locally, I don't think using a constant there should be a good idea as it depends on each composable.

Reply:
Ok we discussed this together and I aggree with your point saying that it is a more global issue than just this PR, therefore I created a specific [issue](https://github.com/polypoly-team/android-app/issues/40) that we can address in a later sprint.